### PR TITLE
Adds a lobby final countdown.

### PR DESCRIPTION
**Pull request by pointer-to-null**
## About the PR 

Setting a round start time to 10 seconds (default value) when all players in the lobby pressed the "Ready Up" button.

The final countdown duration can be changed by using cvar.

https://user-images.githubusercontent.com/91910481/147684742-ea1a4be2-e0a5-420e-9515-2a3856069aab.mp4

**Changelog**

:cl:
- add: Added a final countdown in the lobby when all players are ready.

### CODE REVIEW

File: Content.Server/GameTicking/GameTicker.Lobby.cs [hunk 1]
```diff
@@ -19,6 +19,9 @@ public partial class GameTicker
         [ViewVariables]
         private TimeSpan _roundStartTime;
 
+        [ViewVariables]
+        private readonly TimeSpan _finalCountdownTime = new TimeSpan(0, 0, 10);
```
Comment (mirrorcult) [inline Content.Server/GameTicking/GameTicker.Lobby.cs:23]: should be a cvar imo

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Content.Server/GameTicking/GameTicker.Lobby.cs (hunk 1 above)
Comment (pointer-to-null) [reply Content.Server/GameTicking/GameTicker.Lobby.cs]: Done.

**Comment by Zumorica** [comment]
I don't like this, honestly. The round start timer is long on purpose so people can join the server during it and ready up.

**Comment by pointer-to-null** [comment]
> I don't like this, honestly. The round start timer is long on purpose so people can join the server during it and ready up.

In some cases admin can control this by using delay/pause. But yea, if there is no admin then this not going to work.

So, there are two options here: 
1. Scrap this PR. 
2. I can make this feature toggleable through CVar. And it will be off by default.

Both are fine with me.

**Comment by PaulRitter** [comment]
ngl, i agree with vera & pjb. i dont see the need

**Comment by metalgearsloth** [comment]
If a fork ever asks for it we'll have a reference PR.

**Pull request closed without merging**